metrics_utility/management/commands/build_report.py

```diff
@@ -25,27 +36,37 @@ class Command(BaseCommand):
 
     help = 'Gather Automation Controller billing data'
 
-    def add_arguments(self, parser):
-        parser.add_argument(
-            '--month',
-            dest='month',
-            action='store',
-            help='Month the report will be generated for, with format YYYY-MM. '
-            "If this params isn't provided, previou month report will be"
-            "generated if it doesn't exists already.",
+    def __init__(self):
```

Comment:
When redefining the constructor, don't forget to call the constructor of the base class too...